logstash-core/benchmarks/src/main/resources/log4j2-with-script.properties

```diff
@@ -0,0 +1,60 @@
+status = error
+name = LogstashPropertiesConfig
+
+#appender.console.type = Console
+#appender.console.name = plain_console
+#appender.console.layout.type = PatternLayout
+#appender.console.layout.pattern = [%d{ISO8601}][%-5p][%-25c]%notEmpty{[%X{pipeline.id}]} %m%n
+#appender.console.avoid_pipelined_filter.type = ScriptFilter
+#appender.console.avoid_pipelined_filter.script.type = Script
+#appender.console.avoid_pipelined_filter.script.name = filter_no_pipelined
+#appender.console.avoid_pipelined_filter.script.language = JavaScript
+#appender.console.avoid_pipelined_filter.script.value = ${sys:ls.pipeline.separate_logs} == false || !(logEvent.getContextData().containsKey("pipeline.id"))
```

Comment:
Does uncommenting this block to enable the filter have any effect on the performance?

I think there might be 3 tests - no scripting in `log4j` files, scripting with `separate_logs = false` and scripting with `separate_logs = true`.